test/fixtures/virtual-module/unplugin.js

```diff
@@ -1,4 +1,4 @@
-const { createUnplugin } = require('unplugin')
+const { createUnplugin } = require('../../../')
```

Comment:
This test should have detected this bug in advance, but `require('unplugin')` resolve to an older version (v1.12.2)